scala-thoughts.md

```diff
@@ -0,0 +1,7 @@
+Overall, Scala was comfortable to use because of its similarities to Java. Basic concepts such as iteration and conditionals stayed consistent. Defining variables was also fine, though Python did condition me to keep forgetting to put in val and var keywords before declaring a variable. 
```

Comment:
Yes, I agree with everything. And while I like Scala, I do think that it ends up being a jack of all trades and a master of none.